### ISSUE TITLE
Version is not being correctly read from pubspec.yaml

### ISSUE THREAD

**Issue by cpswan** [atsign-foundation/at_server#389]
**Describe the bug**

Version is being shown as `null` rather than the correct version from pubspec.yaml

```
8fe461b7-9d18-5ccf-9e3b-9ffe4cd09571_secondary.1.b9fsfptnr94p@hornet-10    | INFO|2021-11-09 20:25:24.362967|AtSecondary
Server|Secondary server started on version : null on root server : root.atsign.org
```

Same happens on staging:

```
0bd4703d-abe2-595e-b9f4-834421acabed_secondary.1.vxxpvmj3nbd6@buzz-04    | INFO|2021-11-10 22:36:03.917967|AtSecondaryServer|Secondary server started on version : null on root server : root.atsign.wtf
```

**To Reproduce**
1. Pick a secondary
2. Run `sudo docker service logs $SECONDARY | grep version`

**Expected behavior**
The secondary should read its version number from pubspec.yaml

**Additional context**

For staging we imprint the github run number into the pubspec.yaml:

```
sed -i "s/version\:.*/&+gha${{ github.run_number }}/" pubspec.yaml
```

I've verified that /atsign/pubspec.yaml exists in the container.

**Comment by cpswan** [comment]
@kalluriramkumar has pointed out that this is caused by mounting /atsign rather than /atsign/storage (as we do in dess) meaning that the pubspec.yaml can't be read from /atsign.

#296 was raised to address this issue, and now becomes more of a priority to get fixed.

**Comment by ksanty** [comment]
Linked dependency to issue #296

**Comment by cpswan** [comment]
Contingent work in #296 isn't complete, so this needs to continue in PR26

**Comment by cpswan** [comment]
Now fixed by work for #296
